src/prompt.ts

```diff
@@ -161,13 +163,15 @@ async function runOrReviseFlow(
         value: 'revise',
         hint: 'Give feedback via prompt and get a new result',
       },
+      { label: '📋 Copy', value: 'copy', hint: 'Copy the generated script' },
```

Comment:
Would be great to make clear where it is being copied to
```suggestion
      { label: '📋 Copy', value: 'copy', hint: 'Copy the generated script to your clipboard' },
```